src/vscode-cosmosdb.api.d.ts

```diff
@@ -45,6 +45,7 @@ export interface DatabaseAccountTreeItem extends AzureDatabasesTreeItem {
      */
     azureData?: {
         accountName: string;
+        accountId?: string;
```

Comment:
I would add accountId in the cosmos case, too - so that we don't have to make this optional and add docs explaining why it only works for postgres